dialog/file.go

```diff
@@ -233,7 +233,28 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 		}
 	})
 
+	newFolderButton := widget.NewButtonWithIcon("", theme.FolderNewIcon(), func() {
+		newFolderEntry := widget.NewEntry()
+		ShowForm("New Folder", "Create Folder", "Cancel", []*widget.FormItem{
+			{
+				Text:   "Name",
+				Widget: newFolderEntry,
+			},
+		}, func(s bool) {
+			if !s || newFolderEntry.Text == "" {
+				return
+			}
+
+			createFolderErr := os.MkdirAll(filepath.Join(f.dir.Path(), newFolderEntry.Text), 0700)
+			if createFolderErr != nil {
```

Comment:
It would be good if we could log the error using `fyne.LogError`.

Reply:
thanks ! missed that one out.